### PR TITLE
Add loading icons when loading packages

### PR DESCRIPTION
**Pull request by PoignardAzur**
image

### CODE REVIEW

File: lib/ExecutionControlEpic/ControlPanelFeature/Containers/Panel.js [hunk 1]
```diff
@@ -49,6 +49,7 @@ export default compose(
   connect(
     (state: State) => ({
       packagePanels: getPackagePanelsFromState(state),
+      isLoading: state.ongoingPackageRefreshCount !== 0,
```
Comment (alanzanattadev) [inline lib/ExecutionControlEpic/ControlPanelFeature/Containers/Panel.js]: ongoingPackageRefreshCount shouldn't be in the global state

File: lib/ProjectSystemEpic/PackageFeature/Epics/Packages.js [hunk 2]
```diff
@@ -3,14 +3,21 @@
...
+  plugins: Array<Plugin>,
+) => Rx.Observable<Array<Package>>;
+
+const packagesEpic = (findPackages: packageFinder) => (action$: Observable) => {
   return action$
     .ofType("REFRESH_PACKAGES")
-    .groupBy(a => a.payload.rootPath, null, () => Rx.Observable.timer(3000))
-    .map(g => g.buffer(Rx.Observable.timer(3000)))
+    .groupBy(a => a.payload.rootPath, null, () => Rx.Observable.timer(300))
+    .map(g => g.buffer(Rx.Observable.timer(300)))
```
Comment (alanzanattadev) [inline lib/ProjectSystemEpic/PackageFeature/Epics/Packages.js]: why going from 3000 to 300 ?

Comment (PoignardAzur) [reply lib/ProjectSystemEpic/PackageFeature/Epics/Packages.js]: Two reasons:

- Most of these events happen very close to each other, so having 3s of buffering is overkill.

- While the 3s delay isn't too visible when starting Atom, it's very visible when reloading a package (eg saving changes in package.json).

File: lib/ExecutionControlEpic/ControlPanelFeature/Containers/Panel.js (hunk 1 above)
Comment (PoignardAzur) [reply lib/ExecutionControlEpic/ControlPanelFeature/Containers/Panel.js]: I'm reluctant to modify `PackagesReducer`, which is currently a Map<>; I'd have to either replace it by or move it into an object with the `isLoading` member.

Comment (alanzanattadev) [reply lib/ExecutionControlEpic/ControlPanelFeature/Containers/Panel.js]: You should then create a "globalState" attribut in which you put this attribut, it's cleaner

File: lib/ProjectSystemEpic/PackageFeature/Epics/Packages.js (hunk 2 above)
Comment (alanzanattadev) [reply lib/ProjectSystemEpic/PackageFeature/Epics/Packages.js]: hmmm ok

File: lib/ExecutionControlEpic/ControlPanelFeature/Containers/Panel.js (hunk 1 above)
Comment (PoignardAzur) [reply lib/ExecutionControlEpic/ControlPanelFeature/Containers/Panel.js]: Mh... maybe a `loadingState` attribute and reducer?

**Comment by PoignardAzur** [comment]
@alanzanattadev Ready to merge.

**Pull request merged**